dataset/copy-test/node/arrow/schema.cypher

```diff
@@ -0,0 +1 @@
+create node table tableTypes (id INT64, column1 INT64, column2 DOUBLE, column3 BOOLEAN, column4 DATE, column5 TIMESTAMP, column6 STRING, PRIMARY KEY (id));
```

Comment:
Let's be more explicit. column1 -> int64Column, column2 -> doubleColumn etc. This will make your tests more readable.